### ISSUE TITLE
Stock price forecasting demo

### ISSUE THREAD

**Issue by justinshenk** [traja-team/traja#79]
Demonstrate Traja with a stock market price forecasting example.

**Comment by justinshenk** [comment]
The model [tests](https://github.com/traja-team/traja/blob/master/traja/tests/test_models.py) provide a good starting point for how to use the API.

**Comment by Justus-M** [comment]
@justinshenk is Traja currently limited to predicting all the input variables, or is there a way to pass specific inputs or select a subset of inputs for prediction?

Ex. for this stock market price forecasting use case you would want to use (at least) volume, open, close and price as inputs, and then predict only the price.

**Comment by Saran-nns** [comment]
Yes,currently Traja doesn't support user specified variable prediction. `dataset.MultiModalDataLoader`, consider all features except 'ID' as time series and forecast/predict all time series variables 'k' steps forward.

Yes, by f:R^n-->R^m where n<<m, we can exploit traja models and ripe better predictive performance. 

A simple work around is to
1) Add arg in `MultiModalDataLoader` to accept list of `target_vars` with `assert output_dim==len(target_vars)` in `generate_dataset` method
2) Override the targets [here](https://github.com/traja-team/traja/blob/659c8e0978042d561f8e13a4b0786d28c2ed7a41/traja/dataset/generator.py#L72)

Thanks for the issue.

**Comment by justinshenk** [comment]
@Justus-M does this look like something you could help with?

**Comment by Justus-M** [comment]
Hi guys, sorry I never followed up on this. I did immediately spend some time looking at the documentation to try and implement this. I couldn't get it to run, and as my Pytorch experience is limited (I'm used to TensorFlow) it just ended up being much more time-consuming than initially expected, so I decided not to continue as I've got a lot on my plate at the moment.

I've uploaded some high resolution (minute level) long term apple stock price data, in case someone from the Traja team would like to test it themselves:
https://drive.google.com/file/d/15Lhx9Txmxz7xvM0rFJ4HvMBwkUvdwyhB/view?usp=sharing

**Comment by justinshenk** [comment]
Thanks in any case, Justus, for the update and sharing the data!
